server/src/common/actions/effectifs.actions.ts

```diff
@@ -208,3 +177,212 @@ export const addEffectifComputedFields = (organisme: Organisme): Effectif["_comp
     },
   };
 };
+
+export async function getEffectifForm(effectifId: ObjectId): Promise<any> {
+  const effectif = await effectifsDb().findOne({ _id: effectifId });
+  return buildEffectifResult(effectif);
+}
+
+export async function updateEffectifFromForm(effectifId: ObjectId, body: any): Promise<any> {
+  // eslint-disable-next-line @typescript-eslint/no-unused-vars
+  const { inputNames, ...data } = body; // TODO JOI (inputNames used to track suer actions)
```

Comment:
Pas hyper clair l'intention et la raison ici. Au passage : 

```suggestion
  // eslint-disable-next-line @typescript-eslint/no-unused-vars
  const { inputNames, ...data } = body; // TODO JOI (inputNames used to track user actions)
```

Reply:
+1, y'a un inputName qu'on extrait. Je ne sais même pas si c'est utilisé.

Reply:
Au moins le commentaire avec la typo indique bien que le code est probablement à revoir :upside_down_face: